Wave/Wave/WaveViewController.swift

```diff
@@ -22,8 +22,16 @@ final public class WaveViewController: UIViewController {
         return tableView
     }()
     
+    func dismiss() {
+        dismissViewControllerAnimated(true, completion: nil)
+    }
+    
     override public func loadView() {
         super.loadView()
+        let doneButton = UIBarButtonItem(barButtonSystemItem: .Done, target: self, action: #selector(dismiss))
```

Comment:
Why store this in a let if all you're doing with it is making it the right bar button?